### ISSUE TITLE
[Feature request] Add remaining currencies to crafting calculator shopping list

### ISSUE THREAD

**Issue by sho-87** [gw2efficiency/issues#1411]
Lets say I want to craft an item (like https://gw2efficiency.com/crafting/calculator/a~0!b~0!c~0!d~1-82196) with use own materials selected

The shopping list will tell me the remaining number of items I need to purchase based on my owned materials.

For items purchased using currencies it lists the full currency cost of each item, but it doesn't take into account the number of those currencies I currently have.

For example, it lists Legendary War Insight as costing 1095 WvW Skirmish Claim Tickets in total. It would be nice if instead of the total amount, it shows the number of _remaining_ claim tickets I need to collect to purchase that item.

I feel like that would be more helpful to know what else I need to gather to craft an item. Because those currencies are often gated by play time and not gold, a remaining amount is a more useful indicator of how close I am to completing an item

**Comment by queicherius** [comment]
Duplicate of https://github.com/gw2efficiency/issues/issues/64, right now these currencies are not calculated at all, we just show the current merchant prices
